ldap/servers/slapd/libglobs.c

```diff
@@ -4997,6 +5002,38 @@ config_set_reservedescriptors(const char *attrname, char *value, char *errorbuf,
     return retVal;
 }
 
+int
+config_set_num_listeners(const char *attrname, char *value, char *errorbuf, int apply)
+{
+    int retVal = LDAP_SUCCESS;
+    long nValue = 0;
+    int minVal = 1;
+    int maxVal = 4;
+    char *endp = NULL;
+    slapdFrontendConfig_t *slapdFrontendConfig = getFrontendConfig();
+
+    if (config_value_is_null(attrname, value, errorbuf, 0)) {
+        return LDAP_OPERATIONS_ERROR;
+    }
+
+    errno = 0;
+    nValue = strtol(value, &endp, 0);
+    if (*endp != '\0' || errno == ERANGE || nValue < minVal || nValue > maxVal) {
+        nValue = (nValue < minVal) ? minVal : maxVal;
+        slapi_create_errormsg(errorbuf, SLAPI_DSE_RETURNTEXT_SIZE,
```

Comment:
message should rather mention liseners than conntable split

Reply:
IMHO you should rather reference explicitly the attribute name
 (i.e "%s: invalid value \"%s\", " CONFIG_NUM_LISTENERS_ATTRIBUTE 
        " must range from %d to %d. ""%s ",  ...